templates/default.nginx_http.conf.template

```diff
@@ -15,6 +15,12 @@ log_format  custom  '$http_x_forwarded_for - $remote_user [$time_local] "$reques
 # workaround for the way we're including conf https://serverfault.com/questions/729128/overriding-nginx-access-log-directive-duplicate-log-entries
 access_log  off;
 
+# Don't send nginx version number
```

Comment:
Is this duplicated in https://github.com/dockstore/compose_setup/pull/140 ?

Reply:
Yes, changes from #140 were included to avoid a merge conflict